docs/TooManyRequests.md

```diff
@@ -0,0 +1,18 @@
+
+
+# TooManyRequests
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**type** | **String** | A link to the error documentation. |  [optional]
```

Comment:
just to confirm: type is just a link to the documentation?

Reply:
Yes, in API error responses `type` always links to the docs